services/openshift/scripts/get_ip_address

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+function get_ip_address
+{
+    ip=`ip addr show eth1 | awk 'NR==3 {print $2}' | cut -f1 -d /`
```

Comment:
I would suggest to make this ip as local variable because we are sourcing this script in multiple files and it might case issue.
```
local ip=`ip addr show eth1 | awk 'NR==3 {print $2}' | cut -f1 -d /`
```